flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/runtime/batch/sql/DynamicFilteringITCase.java

```diff
@@ -48,9 +48,7 @@ public class DynamicFilteringITCase extends BatchTestBase {
     private Catalog catalog;
 
     static Stream<Arguments> parameters() {
-        return Stream.of(
-                Arguments.of(BatchShuffleMode.ALL_EXCHANGES_BLOCKING),
-                Arguments.of(BatchShuffleMode.ALL_EXCHANGES_PIPELINED));
+        return Stream.of(Arguments.of(BatchShuffleMode.ALL_EXCHANGES_BLOCKING));
```

Comment:
Is it possible that we introduce a parameter pair (scheduler-type, exchange-type), including
[(Default, ALL_EXCHANGES_PIPELINED), (Default, ALL_EXCHANGES_BLOCKING), (AdaptiveBatch, ALL_EXCHANGES_BLOCKING)]?
Since the scheduler type also needs to be part of the cluster config, this may happen in a test base.

Reply:
I noticed a problem (FLINK-30815) of SQL tests which makes it hard to do this change. 
Therefore, I think it's fine to test blocking exchanges by default because this is how Flink batch is used in production.

Reply:
DynamicFilteringITCase, MultipleInputCreationTest and MultipleInputITCase are reworked so that they now keep testing both ALL_EXCHANGES_PIPELINED and ALL_EXCHANGES_BLOCKING.